src/components/blocks/categoriesBlock.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./blocks.module.scss";
+import GameCategory from "@/components/gameCategoryCard/gamesCategory";
+import { CategoryInfo, categoryInfos } from "@/categoriesInfo";
+
+interface BlockName {
+  blockName: string;
+}
+
```

Comment:
use correct naming/ better IBlockNameProps

Reply:
naming of props interfaces all over the project set to that template you mentioned